### PR TITLE
Elaborated on how to perform the setup

### PR DESCRIPTION
**Pull request by S0urC10ud**
Reading these additions makes it easier for newbies to get started.

I had no idea how to get the access token as I am quite new to Matrix so I think this will help many others out as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,8 +42,23 @@ Note: 'localhost' as 'matrixserver' (in cfg.json) wouldn't work because of docke
 
 # Get started
 1. Create a bot user.
-2. Adjust the config file (cfg.json) to make it work with your matrix server.
-3. Invite your bot into a private room, it will join automatically.<br>
+2. Get an access token to your Matrix-Server: 
+`curl -XPOST -d '{"type":"m.login.password", "user":"@mailBotUsername:your-domain.com", "password":"mailbotPassword"}' "https://matrix.your-domain.com/_matrix/client/r0/login"`
```
Comment (JojiiOfficial) [inline README.md:46]: Could you make this a multi line code block? Looks better in my opinion.

Comment (S0urC10ud) [reply README.md]: yes, you are right

**Review by S0urC10ud — COMMENTED** [review]
CURL-access-token-command: change to multi-line comment

**Review by JojiiOfficial — APPROVED** [review]

**Comment by JojiiOfficial** [comment]
Thanks

**Pull request merged**